Base/home/anon/WindowManager.ini

```diff
@@ -1,6 +1,6 @@
 [Screen]
-Width=2560
-Height=1440
+Width=1920
```

Comment:
Let's keep the default resolution at 1024x768 so it fits on most screens on startup.

Reply:
Ok. Also I need to undo some other settings in there.